examples/next-openai/app/stream-object/page.tsx

```diff
@@ -0,0 +1,64 @@
+'use client';
+
+import { StreamableValue, useStreamableValue } from 'ai/rsc';
+import { useState } from 'react';
+import { generateNotifications } from './actions';
+import { PartialNotification } from './schema';
+
```

Comment:
add this:
// Force the page to be dynamic and allow streaming responses up to 30 seconds
export const dynamic = 'force-dynamic';
export const maxDuration = 30;

Reply:
thanks yes